nmdc_schema/migration_recursion.py

```diff
@@ -123,8 +124,11 @@ def main(schema_path, input_path, output_path, salvage_prefix, migrator_name):
             continue
 
     if len(migrators) == 0:
-        logger.info("No valid migrators specified. Will perform default migration")
-        # exit()
+        # logger.info("No valid migrators specified. Will perform default migration")
```

Comment:
If you are keeping this only so you can easily restore it (and not so you can read it when reading the rest of the code), I'd prefer it be deleted. It will still be available in the Git history.